app/models/RegulatorsAndDocuments.scala

```diff
@@ -57,3 +57,13 @@ object CharityOtherRegulatorDetails {
   override def toString: String = "charityOtherRegulatorDetails"
 
 }
+
+case class WhyNotRegisteredWithCharity(reason:String)
```

Comment:
related to above comments, after using Form[String] this class might not be required.